src/schema/sources.ts

```diff
@@ -1095,10 +1101,25 @@ const addNewSourceMember = async (
   con: DataSource | EntityManager,
   member: Omit<DeepPartial<SourceMember>, 'referralToken'>,
 ): Promise<void> => {
+  const referralToken = randomUUID();
```

Comment:
Existing tokens are as is, new ones will be UUID, it does not really change anything DB wise and length wise its almost the same (urls are shortened either way)